hack/util.sh

```diff
@@ -74,9 +74,9 @@ function start_os_server {
     ps -ef | grep openshift
     echo "[INFO] Starting OpenShift server"
     sudo env "PATH=${PATH}" OPENSHIFT_PROFILE=web OPENSHIFT_ON_PANIC=crash openshift start \
-	 --master-config=${MASTER_CONFIG_DIR}/master-config.yaml \
```

Comment:
That change is not relevant to the topic of this PR. If somebody will be `git blame` this portion of code, your name will show up :-)